test/functional/sqlsrv/0065.phpt

```diff
@@ -1,46 +1,41 @@
 --TEST--
 inserting and retrieving UTF-8 text.
 --SKIPIF--
-<?php require('skipif.inc'); ?>
+<?php require('skipif_versions_old.inc'); ?>
 --FILE--
 <?php
 
-sqlsrv_configure( 'WarningsReturnAsErrors', 0 );
-sqlsrv_configure( 'LogSeverity', SQLSRV_LOG_SEVERITY_ALL );
+sqlsrv_configure('WarningsReturnAsErrors', 0);
+sqlsrv_configure('LogSeverity', SQLSRV_LOG_SEVERITY_ALL);
 
 // For testing in Azure, can not switch databases
-require( 'MsCommon.inc' );
-$c = Connect();
-
-if( $c === false ) {
-    die( print_r( sqlsrv_errors(), true ));
-}
-
-$s = sqlsrv_query( $c, "IF OBJECT_ID('utf8test', 'U') IS NOT NULL DROP TABLE utf8test" );
-
-// test a varchar, nvarchar non max, and nvarchar max
-$s = sqlsrv_query( $c, "CREATE TABLE utf8test (c1 varchar(100), c2 nvarchar(100), c3 nvarchar(max))");
-if( $s === false ) {
-    die( print_r( sqlsrv_errors(), true ));
+require_once('MsCommon.inc');
+$c = AE\connect();
+
+$columns = array(new AE\ColumnMeta('varchar(100)', 'c1'),
+                 new AE\ColumnMeta('nvarchar(100)', 'c2'),
+                 new AE\ColumnMeta('nvarchar(max)', 'c3'));
+$stmt = AE\createTable($c, "utf8test", $columns);
```

Comment:
create a `$tableName` variable to hold "utf8test"; use this variable for querying anything from this table

Reply:
that was what I did initially but the variable didn't seem to work with the <<< operator

Reply:
would be nice for all tests to be consistent and uses a table name variable. Perhaps get rid of the <<< operator? also, putting variables into single quotes won't work either (line 94)